frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -128,13 +128,45 @@ const HostSummary = ({
     </div>
   );
 
+  // TODO: should these strings be imported from an external constants file? This would be repeating
+  // functionality that exists elsewhere (i.e., ManageHosts page) – encapsulate it all in a more
+  // general location (statuscell/constants?)
```

Comment:
Yes you are thinking along the right path here. But I would extend that and DRY up this code by creating a status component that also can encapsulate the rendering markup and the logic and constants to display the correct tooltip text.